### PR TITLE
Add a StringOption example to the README

### PR DESCRIPTION
**Pull request by warpling**
I couldn't find any syntactic examples of how to use this super useful feature so I tried to add one. Let me know if anything needs *tweaking* ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,8 @@ public struct ExampleTweaks: TweakLibraryType {
 	public static let colorTint = Tweak("General", "Colors", "Tint", UIColor.blue)
 	public static let marginHorizontal = Tweak<CGFloat>("General", "Layout", "H. Margins", defaultValue: 15, min: 0)
 	public static let marginVertical = Tweak<CGFloat>("General", "Layout", "V. Margins", defaultValue: 10, min: 0)
-	public static let featureFlagMainScreenHelperText = Tweak("Feature Flags", "Main Screen", "Show Body Text", true)
+	public static let font = Tweak<StringOption>.stringsList("General", "Layout", "Font", options: ["AvenirNext", "Helvetica", "SanFrancisco"])
```
Comment (ShivaHuang) [inline README.md]:
```suggestion
	public static let font = Tweak("General", "Layout", "Font", options: ["AvenirNext", "Helvetica", "SanFrancisco"])
```
After PR https://github.com/Khan/SwiftTweaks/pull/120 merged, the old initializer for `StringOption` was deprecated. You can now initialize a Tweak for `StringOption` like other types.

Comment (warpling) [reply README.md]: Great!

**Comment by warpling** [comment]
All good now I think :)

**Comment by ShivaHuang** [comment]
I think you didn't apply the change. 🙈

**Comment by warpling** [comment]
@ShivaHuang woah my bad—I assumed I did. That could be made clearer on GH's end I guess 🙄 
Not sure I still can? I managed to merge one I think…

File: README.md (hunk 1 above)
Comment (warpling) [reply README.md]: Ah I resolved the wrong one.

**Review by bryanjclark — APPROVED** [review]
Thanks for adding documentation!

**Pull request merged**